src/hypermedia-constants.js

```diff
@@ -366,7 +366,7 @@ export const Classes = {
 			recommended: 'recommended',
 			enforced: 'enforced',
 			showClock: 'show-clock',
-			automaticZero: 'automatic-zero'
+			automaticZero: 'uselatelimit'
```

Comment:
Is this right? Looks like the [serializer](https://search.d2l.dev/search?project=lms&full=%22automatic-zero%22&defs=&refs=&path=&hist=&type=&xrd=&si=full&si=full) still uses `automatic-zero`?

Reply:
Good question. I'm updating it here! https://github.com/Brightspace/lms/pull/6616/files#diff-3ac4b97bf1fa5b1a1c7096d96a97e57c4a038805fd97ddbd50b790636a1a3426R332